dome9CloudBots/bots/modify_network_security_group_scope_by_port.py

```diff
@@ -0,0 +1,69 @@
+# What it does: Change network security group scope by a given port.
+# Scope can be list of ip addresses with , between. example: 192.168.99.0/24,10.0.0.0/24,44.66.0.0/24
+# Direction can be : source or destination
+# Bot will remove Any from direction
+# Usage: AUTO: modify_network_security_group_scope_by_port <port> <direction> <scope>
+# Example: AUTO: modify_network_security_group_scope_by_port 556 source 10.0.0.0/24,172.16.0.1/32,168.243.22.0/23
+# Limitations: None
+
+from msrestazure.azure_exceptions import CloudError
+from azure.mgmt.network import NetworkManagementClient
+import logging
+
+
+def modify_scope(rule, direction, scope):
+    if len(scope) == 1:
```

Comment:
len(scope) == 1 should be a variable!

Reply:
1 should be a consts

Reply:
e.g.

MUST_HAPPEN = 1
isItHappen = len(scope) == MUST_HAPPEN
if isItHappen:
pass